root_profiler/engine/db.php

```diff
@@ -93,36 +70,32 @@ function query($query = "", $transaction = FALSE)
 			$this->num_queries++;
 			if( $transaction == BEGIN_TRANSACTION && !$this->in_transaction )
 			{
-				$result = mysql_query("BEGIN", $this->db_connect_id);
-				if(!$result)
+				if ($this->dbh->exec("BEGIN") === false)
 				{
 					return false;
 				}
-				$this->in_transaction = TRUE;
+				$this->in_transaction = true;
 			}
 
-			$this->query_result = mysql_query($query, $this->db_connect_id);
+			$this->query_result = $this->dbh->query($query);
 		}
 		else
 		{
```

Comment:
(Pre-existing) surely PHP has an else if construct, saving a layer of indent ...